pkgs/applications/audio/gwc/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkgconfig
+, alsaLib
+, libpulseaudio
+, gtk2
+, hicolor-icon-theme
+, libsndfile
+, fftw
+, perl
+, xdg_utils
+, vorbis-tools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gwc";
+  version = "0.22-04";
+
+  src = fetchFromGitHub {
+    owner = "AlisterH";
+    repo = pname;
+    rev = version;
+    sha256 = "0xvfra32dchnnyf9kj5s5xmqhln8jdrc9f0040hjr2dsb58y206p";
+  };
+
+  buildInputs = [
+    alsaLib
+    libpulseaudio
+    gtk2
+    hicolor-icon-theme
+    libsndfile
+    fftw
+    perl
+    xdg_utils
+    vorbis-tools
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+  ];
+
+  meta = with stdenv.lib; {
+    description = "a gui application to remove noise (hiss, pops and clicks) from audio files";
+    homepage = "http://gwc.sourceforge.net/";
+    license = licenses.gpl1;
```

Comment:
So old school :).

Reply:
Indeed.

Reply:
The source code looks gpl2+ https://github.com/AlisterH/gwc/blob/441d67f3dfabf3a739656ff1b9fb4b546817fbb3/gwc.c#L5-L8